### ISSUE TITLE
Question: When will the SDK Runtime feature be enabled by default?

### ISSUE THREAD

**Issue by canyie** [android/privacy-sandbox-samples#136]
Hello,

This is not a Bug Report, but a question.

I have been interested in the new SDK Runtime feature for at least 2 years. I tried using the platform API (instead of the Jetpack support library because it is unacceptably large, I can handle compatibility issues myself) to load SDKs, and everything worked fine. However, I have to manually turn on the feature by running adb commands, even on the latest Android 16 DP1. Even on the main branch, the feature is still disabled by default: https://cs.android.com/android/platform/superproject/main/+/main:packages/modules/AdServices/sdksandbox/service/java/com/android/server/sdksandbox/SdkSandboxSettingsListener.java;l=49

While the official document says Android 15 includes this feature: https://developer.android.com/about/versions/15/features#privacy-sandbox : 
> Android 15 includes the latest Android Ad Services extensions, incorporating the latest version of the [Privacy Sandbox on Android](https://developer.android.com/design-for-safety/privacy-sandbox). This addition is part of our work to develop technologies that improve user privacy and enable effective, personalized advertising experiences for mobile apps. Our [privacy sandbox page](https://developer.android.com/design-for-safety/privacy-sandbox/program-overview) has more information about the Privacy Sandbox on Android developer preview and beta programs to help you get started.

I have no idea when the feature will be ready for use in production builds. I've searched everywhere but found nothing. Anyone know the ETA of this new feature? Your help will be greatly appreciated.

**Comment by notmariazoe** [comment]
Hi @canyie, and thanks for the question! 

As you correctly pointed out, the SDK Runtime feature is shipped with devices running Android 14 and above, but activating it requires a manual `adb` command. We're in the process of gradually activating the feature across the GMS fleet of devices, aiming to achieve 95%+ activation by end of Q2 2025. 

Production use of the SDK Runtime also requires supporting app and SDK distribution. We're engaging with the distribution ecosystem to extend support across the widest possible array of distribution platforms. We plan on having an update on distribution and production milestones throughout 2025.  

In the meantime, you can test runtime-enabled applications  in a **non local deployment flow** with Google Play closed beta, if you [register interest in SDK Runtime distribution](https://docs.google.com/forms/d/e/1FAIpQLSe_nIooIplKELeTAFlWEgAeisMS96QSMFBMyvG7L3KIEqY4DA/viewform%22).

You mention that you're finding the Jetpack libraries too large for you, and you're handling backwards compatibility by yourself. I'd love to hear more about your experience and use cases, is this still true for you? The Jetpack libraries go beyond backwards compatibility, helping you abstract APIs between processes without using AIDL, or present UI and launch activities from the Runtime. Are you familiar with the latest [Runtime-Enabled SDK developer guide](https://developers.google.com/privacy-sandbox/private-advertising/sdk-runtime/developer-guide)?

If you'd like more information on the latest updates, like new tools and APIs, I recommend you check the latest library releases*, watch [the current sample](https://github.com/android/privacy-sandbox-samples/tree/main/PrivacySandboxKotlin) to be notified of changes, or check our [documentation hub](https://goo.gle/sdk-runtime). You can also join the [SDK Runtime mailing list](https://groups.google.com/g/sdkrt-announce) to receive the latest news and announcements.

\*Latest releases for the [Shim](https://developer.android.com/jetpack/androidx/releases/privacysandbox-tools), [Backwards compatibility](https://developer.android.com/jetpack/androidx/releases/privacysandbox-sdkruntime), [Remote UI](https://developer.android.com/jetpack/androidx/releases/privacysandbox-ui), and [Activity support](https://developer.android.com/jetpack/androidx/releases/privacysandbox-activity) Jetpack libraries.

**Comment by notmariazoe** [comment]
Of course, happy to help! I'm interested in learning about your specific use-case, if and when you'd like to share about it. 
I'll close this issue for now, but feel free to reach out!
